waku/waku_api/rest/lightpush/openapi.yaml

```diff
@@ -76,7 +76,7 @@ components:
     PushRequest:
       type: object
       properties:
-        pusbsubTopic:
+        pubsubTopic:
```

Comment:
Great catch!
Notice that we must correct it here as well:
   In  Waku-org/waku-rest-api repository.